product_attribute_description/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Product Attribute Description",
+    "version": "15.0.1.0.0",
+    "category": "Website",
+    "author": "ERP Harbor Consulting Services,"
```

Comment:
Add Nitrokey GmbH too.

Reply:
Hello @jans23,

I've already added and you can check this URL for the same 

https://github.com/OCA/product-attribute/blob/014744062386d7c00950a59081fcff443adcb407/product_attribute_description/__manifest__.py#L5-L8

Regards,

Dhara Solanki